sirepo/pkcli/admin.py

```diff
@@ -66,7 +66,7 @@ def _link_or_unlink_proprietary_files(sim_type, should_link):
 
     server.init()
     t = feature_config.cfg().proprietary_sim_types
-    if not t:
+    if not t or not t.issubset(feature_config.cfg().sim_types):
```

Comment:
Needs to be the intersection:

```py
t = feature_config.cfg().proprietary_sim_types.intersection(feature_config.cfg().sim_types)
```

Reply:
https://github.com/radiasoft/sirepo/pull/2512/commits/68895ffa7763f0b5edcf67496567e5d871501880